v3iofs/file.py

```diff
@@ -15,21 +15,46 @@
 from http import HTTPStatus
 
 from fsspec.spec import AbstractBufferedFile
-from v3io.dataplane import Client
 
 from .path import split_container
 
 
 class V3ioFile(AbstractBufferedFile):
+    """ File-like operation on V3IO Files """
+
+    def __init__(
```

Comment:
What's the reason for adding explicit `__init__`? Is it just for direct acces to `client`?

Reply:
Yes.  It also explicitly allows others to understand the declared mode, block_size, autocommit behavior, cache_type, and cache_options here, rather than having to go look at the AbstractBuffered file.